env/posix/ocf_env.h

```diff
@@ -647,10 +623,13 @@ struct env_timeval {
 
 uint32_t env_crc32(uint32_t crc, uint8_t const *data, size_t len);
 
-#define ENV_PRIu64 "lu"
-
 unsigned env_get_execution_context(void);
 void env_put_execution_context(unsigned ctx);
 unsigned env_get_execution_context_count(void);
 
+#ifdef UNSTATIC
```

Comment:
is it really necessary to have #ifdef UNSTATIC in this file?

Reply:
removed